charts/loki-distributed/templates/ruler/statefulset-ruler.yaml

```diff
@@ -102,6 +102,9 @@ spec:
             {{- end }}
           resources:
             {{- toYaml .Values.ruler.resources | nindent 12 }}
+        {{- if .Values.ruler.extraContainers }}
+        {{- toYaml .Values.ruler.extraContainers | nindent 8}}
+        {{- end }}
```

Comment:
```suggestion
        {{- with .Values.ruler.extraContainers }}
        {{- toYaml . | nindent 8}}
        {{- end }}
```

Reply:
@zanhsieh thanks, I've updated the commit with your suggestion and have bumped the chart version. The README is already up-to-date, this PR updates the code to match the docs.